### PR TITLE
OpenCV

### PR DESCRIPTION
**Pull request by PaDarochek**

### CODE REVIEW

File: projects/opencv/core.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "-l debug -n 1024 --app-timeout 180"
```
Comment (SweetVishnya) [inline projects/opencv/core.toml:18]: We should check that 180 s is enough for Sydr

File: projects/opencv/build.sh [hunk 2]
```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
```
Comment (SweetVishnya) [inline projects/opencv/build.sh:3]: ISP RAS copyright

File: projects/opencv/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sweetvishnya/ubuntu20.04-sydr-fuzz
+
+RUN apt-get update && apt-get install -y build-essential cmake pkg-config
+
+RUN git clone --depth 1 https://github.com/opencv/opencv.git opencv
```
[the remark this answers is not in the document]
Comment (PaDarochek) [reply projects/opencv/Dockerfile:21]: Done

File: projects/opencv/build.sh [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+################################################################################
+
+# Build targets for libfuzzer
+
+export CC=clang
+export CXX=clang++
+export CFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+export CXXFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+export LDFLAGS="-g -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
```
[the remark this answers is not in the document]
Comment (PaDarochek) [reply projects/opencv/build.sh:24]: Done

File: projects/opencv/build.sh (hunk 2 above)
Comment (PaDarochek) [reply projects/opencv/build.sh:3]: Done

File: projects/opencv/build.sh [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=$install_dir \
+  -DBUILD_SHARED_LIBS=OFF -DOPENCV_GENERATE_PKGCONFIG=ON \
+  -DOPENCV_GENERATE_PKGCONFIG=ON -DOPENCV_FORCE_3RDPARTY_BUILD=ON \
+  -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_opencv_apps=OFF \
+  $SRC
+make -j$(nproc)
+make install
+popd
+
+export CFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
```
[the remark this answers is not in the document]
Comment (PaDarochek) [reply projects/opencv/build.sh:44]: Done

File: projects/opencv/core.toml (hunk 1 above)
Comment (PaDarochek) [reply projects/opencv/core.toml:18]: Done

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**